src/modules/verifiable-credentials/verifiable-credentials-base.service.ts

```diff
@@ -132,6 +140,44 @@ export abstract class VerifiableCredentialsServiceBase {
     );
   }
 
+  /**
+   * @description Sends credentials requested by issuer and returnes either issued credentials or next credentials request
+   *
+   * @param params.vpRequest credentials required to continue exchange
+   * @returns issued credentials or request of additional credentials
+   */
+  public async continueExchange({
+    vpRequest: {
+      challenge,
+      interact: { service },
+    },
+    credentials,
+  }: {
+    vpRequest: VpRequest;
+    credentials: VerifiableCredential<RoleCredentialSubject>[];
+  }) {
+    const requiredPresentation = await this.createVerifiablePresentation(
+      credentials,
+      {
+        challenge,
+      }
+    );
+
+    const {
+      data: { errors, vp: requestedPresentation, vpRequest: nextVpRequest },
+    } = await axios.put<ExchangeResponse>(service[0].serviceEndpoint, {
+      requestedPresentation: requiredPresentation,
```

Comment:
Would it make sense to align the names here?
```suggestion
      requiredPresentation: requiredPresentation,
```

Reply:
Actually, it looks like the right name is just `presentation` as it used in https://github.com/energywebfoundation/ssi/blob/master/apps/vc-api/docs/tutorials/rebeam-tutorial.md#customer-create-the-required-presentation. Is that right?